.circleci/config.yml

```diff
@@ -13,24 +13,28 @@ workflows:
           executor: haskell/ghc-8_2_2
           context: haskell-ci
           binary-cache-uri: ${BINARY_CACHE_URI-"http://hw-binary-cache-us-west-2-a.s3-website-us-west-2.amazonaws.com/archive"}
+          cabal-build-extra: --write-ghc-environment-files=ghc8.4.4+
```

Comment:
This is needed because `doctest` relies on environment files existing in the project directory.  This flag will generate those files in the `v2-build` command so that it will exist by the time `v2-test` is run.